### PR TITLE
Updates for bump

### PR DESCRIPTION
**Pull request by damemi**

### CODE REVIEW

**Comment by damemi** [comment]
/assign @deads2k 
ptal

File: pkg/operator/operatorclient/informers.go [hunk 1]
```diff
@@ -1,27 +1,18 @@
...
-		TargetNamespaceName:                   informers.NewSharedInformerFactoryWithOptions(kubeClient, 10*time.Minute, informers.WithNamespace(TargetNamespaceName)),
-		"kube-system":                         informers.NewSharedInformerFactoryWithOptions(kubeClient, 10*time.Minute, informers.WithNamespace("kube-system")),
-	}
-}
-
-func (i KubeInformersForNamespaces) Start(stopCh <-chan struct{}) {
-	for _, informer := range i {
-		informer.Start(stopCh)
-	}
+func NewKubeInformersForNamespaces(kubeClient kubernetes.Interface) v1helpers.KubeInformersForNamespaces {
```
Comment (deads2k) [inline pkg/operator/operatorclient/informers.go]: I just updated https://github.com/openshift/cluster-kube-apiserver-operator/pull/208 with this

you should delete this

**Comment by damemi** [comment]
This is all covered in https://github.com/openshift/cluster-kube-apiserver-operator/pull/208

**Pull request closed without merging**